davealex/quick-credit
# #165900754 Test and Implement signup API

#### What does this PR do?
Test and Implement signup API endpoint
#### Description of Task to be completed?
Write Unit tests for `/auth/signup` endpoint
Implement endpoint and validate requests
#### How should this be manually tested?
Using Postman, make a request  to the v1 API endpoint: POST `/api/v1/auth/signup` and ensure that the appropriate JSON responses are returned (when passing validation, and when failing validation)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165900754](https://www.pivotaltracker.com/story/show/165900754)
